x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_service_provider/service_provider.test.tsx

```diff
@@ -268,4 +268,25 @@ describe('ServiceProvider component', () => {
       expect(screen.getByText('model-bedrock-xyz')).toBeInTheDocument();
     });
   });
+
+  describe('with alibabacloud-ai-search service', () => {
+    const mockEndpoint = {
+      inference_id: 'alibabacloud-ai-search-1',
+      service: 'alibabacloud-ai-search',
+      service_settings: {
+        service_id: 'service-123',
+        host: 'host-123',
+        workspace: 'default-123',
+      },
+    } as any;
+
+    it('renders the component with endpoint details', () => {
+      render(<EndpointInfo endpoint={mockEndpoint} />);
+
+      expect(screen.getByText('alibabacloud-ai-search-1')).toBeInTheDocument();
+      expect(screen.getByText('service-123')).toBeInTheDocument();
+      expect(screen.getByText('host-123')).toBeInTheDocument();
+      expect(screen.getByText('default-123')).toBeInTheDocument();
```

Comment:
You use a wrong component in your tests. The following should work, tested in locally. Please add more assertions if needed.
```suggestion
      render(<ServiceProvider providerEndpoint={mockEndpoint} />);

      expect(screen.getByText('AlibabaCloud AI Search')).toBeInTheDocument();
      const icon = screen.getByTestId('table-column-service-provider-alibabacloud-ai-search');
      expect(icon).toBeInTheDocument();
```

To run the tests locally:
```
 node scripts/jest --config=x-pack/plugins/search_inference_endpoints/jest.config.js --runInBand --coverage=false --passWithNoTests
```